docs/source/udf/buffer_types.py

```diff
@@ -0,0 +1,38 @@
+from libertem.udf import UDF
+
+class test_buffer(UDF):
+
+	def get_results_buffer(self):
+		"""
+		Initialize a buffer of different types
+
+		The first buffer "single_buffer" specifies a buffer of
+		dimension (16, 16) with entries of type float32
+
+		The second buffer "sig_buffer" specifies a buffer whose
+		first component dimension is increased by 2 from that of
+		signal dimension
+
+		The third buffer "nav_buffer" specifies a buffer whose
```

Comment:
I think for these three buffers, it would help to give example shapes. If for example my dataset has shape `(14, 14, 32, 32)` with two signal dimensions (`nav` would be `(14, 14)`, sig `(32, 32)`), how would the concrete shapes represented by each buffer be like?

Also, what exactly do you mean by "increased" here? It may be misunderstood to mean the existing dimensions are somehow changed, where in reality, new dimensions are introduced.